src/zoid/button/prerender.jsx

```diff
@@ -42,11 +44,11 @@ export function PrerenderedButton({ nonce, onRenderAuth, props } : PrerenderedBu
         }
     };
 
+    
     return (
         <html>
             <body>
-                {/* $FlowFixMe */}
-                <AuthButton { ...props } onClick={ handleClick } />
+                <SpinnerPage nonce={ nonce } />
```

Comment:
This does not seem correct. Why render spinner page inside a button iframe? Why not continue to do pre-render of the button?

Reply:
once we have the contect ready from ice it delays some time in that mean time instead of showing just the button without any text just the spinner before loading it.